__tests__/jest.setup.js

```diff
@@ -3,6 +3,12 @@ import '@testing-library/jest-dom';
 // fix https://github.com/testing-library/dom-testing-library/releases/tag/v7.0.0
 import MutationObserver from '@sheerun/mutationobserver-shim';
 
+jest.mock('next/config', () => () => ({
```

Comment:
We have`facegame/__tests__/jest.setEnvVars.js` and there we can set env vars, no need to mock 'next/config'